### PR TITLE
[15.0] stock_location_lockdown: Migration to 15.0

### PR DESCRIPTION
**Pull request by BT-dmontull**

### CODE REVIEW

**Comment by BT-dmontull** [comment]
@hparfr @LoisRForgeFlow Could you check this PR to be able to merge? I saw that you already did it in the previous version: [#1031](https://github.com/OCA/stock-logistics-warehouse/pull/1031)
Thanks.

**Review by rousseldenis — APPROVED** [review]

File: stock_location_lockdown/models/stock_quant.py [hunk 1]
```diff
@@ -16,8 +16,11 @@ def check_location_blocked(self):
             if record.location_id.block_stock_entrance:
                 raise ValidationError(
                     _(
-                        "The location %s is blocked and can "
-                        "not be used for moving the product %s"
+                        "The location %(location)s is blocked and can "
+                        "not be used for moving the product %(product)s"
```
Comment (BT-aestebanez) [inline stock_location_lockdown/models/stock_quant.py:20]: Don't you have to change the pot file?

**Review by BT-aestebanez — CHANGES_REQUESTED** [review]

**Comment by BT-aestebanez** [comment]
Please improve the name of the PR

File: stock_location_lockdown/models/stock_quant.py (hunk 1 above)
Comment (rousseldenis) [reply stock_location_lockdown/models/stock_quant.py:20]: @BT-aestebanez Usually, the translation updates can be done after merge at https://translation.odoo-community.org/

Comment (BT-aestebanez) [reply stock_location_lockdown/models/stock_quant.py:20]: Thanks for the information :)

**Review by BT-aestebanez — APPROVED** [review]

**Pull request merged**